src/components/measureGroups/MeasureGroupWarningDialog.tsx

```diff
@@ -0,0 +1,160 @@
+import React from "react";
+import {
+  Dialog,
+  DialogContent,
+  DialogActions,
+  DialogTitle,
+  Divider,
+  IconButton,
+  Typography,
+} from "@mui/material";
+import { Button } from "@madie/madie-components";
+import CloseIcon from "@mui/icons-material/Close";
+import { makeStyles } from "@mui/styles";
+
+const useStyles = makeStyles({
+  row: {
+    display: "flex",
+    flexDirection: "row",
+  },
+  paper: {
+    position: "relative",
+    overflow: "visible",
+    marginTop: -20,
+  },
+  dialogTitle: {
+    display: "flex",
+    flexDirection: "row",
+    justifyContent: "space-between",
+    alignItems: "center",
+    padding: "24px 32px",
+  },
+  title: {
+    fontFamily: "Rubik",
+    fontSize: 24,
+    padding: 0,
+  },
+  close: {
+    color: "#242424",
+  },
+  info: {
+    fontSize: 15,
+    fontWeight: 300,
+    fontFamily: "Rubik",
+  },
+  subText: {
+    fontSize: 13.5,
+    fontWeight: "bold",
+    marginTop: 13,
+  },
+  bottomInfo: {
+    fontSize: 12,
+    color: "red",
+    fontWeight: 300,
+    marginTop: 20,
+  },
+
+  dividerBottom: {
+    marginTop: 5,
+  },
+  actionsRoot: {
+    padding: 16,
+    "& >:not(:first-of-type)": {
+      marginLeft: 16,
+    },
+  },
+});
+
+export interface MeasureGroupsWarningDialogProps {
+  modalType?: string;
+  measureGroupNumber?: number;
+  open?: boolean;
+  onClose?;
+  onSubmit?;
+}
+
+export default function MeasureGroupsWarningDialog(
+  props: MeasureGroupsWarningDialogProps
+) {
+  const { modalType, measureGroupNumber, open, onClose, onSubmit } = props;
+  const classes = useStyles();
+
+  const scoringChangeDialogText = {
+    title: "Change Scoring?",
+    info: "Your Measure Scoring is about to be saved and updated based on these changes. Any expected values on your test cases will be cleared for this measure.",
+    subText: "Are you sure you want to Save Changes?",
+    bottomInfo: "This action cannot be undone.",
+    dataTestId: "update-measure-group-scoring",
+  };
+
+  const deleteMeasureGroupDialogText = {
+    title: "Delete Measure Group?",
+    info: `Measure Group ${
+      measureGroupNumber + 1
+    } will be deleted. Are you sure you want to delete this measure group ?`,
```

Comment:
Looks like an extra space between group and the question mark.

Reply:
Thanks Joe, I have fixed it after your suggestion